content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -62,6 +62,83 @@ desired configuration property:
 For example, `exporter_otlp_endpoint` would convert to
 `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`.
 
+## Python-specific Environment Variables
+
+### Excluded URLs
+
+Comma-separated regexes representing which URLs to exclude across all
+instrumentations:
+
+- OTEL_PYTHON_EXCLUDED_URLS
+
+Or exclude URLs from specific instrumentations:
+
+- OTEL_PYTHON_DJANGO_EXCLUDED_URLS
+- OTEL_PYTHON_FALCON_EXCLUDED_URLS
+- OTEL_PYTHON_FASTAPI_EXCLUDED_URLS
+- OTEL_PYTHON_FLASK_EXCLUDED_URLS
+- OTEL_PYTHON_PYRAMID_EXCLUDED_URLS
+- OTEL_PYTHON_REQUESTS_EXCLUDED_URLS
+- OTEL_PYTHON_STARLETTE_EXCLUDED_URLS
+- OTEL_PYTHON_TORNADO_EXCLUDED_URLS
+- OTEL_PYTHON_URLLIB_EXCLUDED_URLS
+- OTEL_PYTHON_URLLIB3_EXCLUDED_URLS
+
+Examples:
+
+- `export OTEL_PYTHON_EXCLUDED_URLS="client/.*/info,healthcheck"`
+- `export OTEL_PYTHON_URLLIB3_EXCLUDED_URLS="client/.*/info"`
+
+### Request Attribute Names
+
+Set to a comma-separated list of attributes names that will be extracted from
+the request object and set as attributes on spans.
```

Comment:
that needs a little bit more explanation, to me it is not clear what this is doing exactly

Reply:
This is how it's mentioned in `opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py`

> Request attributes:
> To extract attributes from Django's request object and use them as span attributes, set the environment variable
> ``OTEL_PYTHON_DJANGO_TRACED_REQUEST_ATTRS`` to a comma delimited list of request attribute names.

Do you have any recommendations on how phrase this to make it more understandable?

Reply:
I am not a python expert but is there some commonality across frameworks? By adding a list of shared attributes it would be much clearer what is possible, otherwise you could link to the framework documentations to give people some context on what they could extract potentially?

Reply:
I would probably just say:

> Comma-separated list of names that will be extracted from
the request object and set as attributes on spans.

Then I think it's pretty clear what's happening, especially with the examplebelow.